src/aap_eda/services/rulebook.py

```diff
@@ -15,6 +15,32 @@
 from aap_eda.core import models
 
 
+def build_source_list(rulesets: list[dict]) -> list[dict]:
+    results = []
+    if rulesets is None:
+        return results
+
+    i = 1
+    for ruleset in rulesets:
+        srcs = ruleset.get("sources", [])
+        for src in srcs:
+            src_record = {}
+            src_record["name"] = f"source_{i}"
```

Comment:
@hsong-rh Can we move this source_ to be constant in the class as DEFAULT_SOURCE_NAME_PREFIX?